tests/test_questions/test_update_question.py

```diff
@@ -51,3 +52,25 @@ def test_increment_total_views_of_questions(self):
             query_update_total_views_of_questions,
             '2'
         )
+
+    def test_database_connection_error(self):
+        """
+        test a user friendly message is returned to a user when database
+        cannot be reached
+        """
+        BaseTestCase().tearDown()
+        CommonTestCases.admin_token_assert_in(
+            self,
+            update_question_mutation,
+            "The database cannot be reached"
+            )
+
+    def test_create_question_without_question_model(self):
```

Comment:
Are we testing whether this is possible or not?

Reply:
Rectified @Bonifase